src/twitch.js

```diff
@@ -32,30 +34,53 @@ Twitch.prototype.makeRequest = function(http) {
 }
 
 /**
-* @description : gets user data from the api
-* @param {String} username : the username we want information from
-* @returns {Promise.<string, Error>} : resolves JSON data or rejects an error
-*/
-Twitch.prototype.getUser = function(username) {
+ * @method getUserId
```

Comment:
yeah, you could just append it to the test file

I was suggesting this so that you could gain some experience with the nock tests (they aren't that complicated, don't worry)